.github/workflows/experimental_build.yml

```diff
@@ -0,0 +1,79 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Experimental build
```

Comment:
Nitpick: What do you think about renaming this workflow to "Specific compiler build"?

Reply:
What about "I'm Feeling Lucky Build"? 😄